pandas/tseries/tests/test_timeseries.py

```diff
@@ -2718,1247 +2706,6 @@ def test_dataframe_dtypes(self):
-
-            with warnings.catch_warnings(record=True):
-                for normalize in (True, False):
-                    for do in offsets:
-                        if isinstance(do, tuple):
-                            do, kwargs = do
-                        else:
-                            do = do
-                            kwargs = {}
-
-                        for n in [0, 5]:
-                            if (do in ['WeekOfMonth', 'LastWeekOfMonth',
-                                       'FY5253Quarter', 'FY5253'] and n == 0):
-                                continue
-                            op = getattr(pd.offsets, do)(n,
-                                                         normalize=normalize,
-                                                         **kwargs)
-                            assert_func(klass([x + op for x in s]), s + op)
-                            assert_func(klass([x - op for x in s]), s - op)
-                            assert_func(klass([op + x for x in s]), op + s)
-
-    def test_overflow_offset(self):
-        # xref https://github.com/statsmodels/statsmodels/issues/3374
-        # ends up multiplying really large numbers which overflow
-
-        t = Timestamp('2017-01-13 00:00:00', freq='D')
-        offset = 20169940 * pd.offsets.Day(1)
-
-        def f():
-            t + offset
-        self.assertRaises(OverflowError, f)
-
-        def f():
-            offset + t
-        self.assertRaises(OverflowError, f)
-
-        def f():
-            t - offset
-        self.assertRaises(OverflowError, f)
-
-    def test_get_duplicates(self):
-        idx = DatetimeIndex(['2000-01-01', '2000-01-02', '2000-01-02',
-                             '2000-01-03', '2000-01-03', '2000-01-04'])
-
-        result = idx.get_duplicates()
-        ex = DatetimeIndex(['2000-01-02', '2000-01-03'])
-        tm.assert_index_equal(result, ex)
-
-    def test_argmin_argmax(self):
-        idx = DatetimeIndex(['2000-01-04', '2000-01-01', '2000-01-02'])
-        self.assertEqual(idx.argmin(), 1)
-        self.assertEqual(idx.argmax(), 0)
-
-    def test_sort_values(self):
-        idx = DatetimeIndex(['2000-01-04', '2000-01-01', '2000-01-02'])
-
-        ordered = idx.sort_values()
-        self.assertTrue(ordered.is_monotonic)
-
-        ordered = idx.sort_values(ascending=False)
-        self.assertTrue(ordered[::-1].is_monotonic)
-
-        ordered, dexer = idx.sort_values(return_indexer=True)
-        self.assertTrue(ordered.is_monotonic)
-        self.assert_numpy_array_equal(dexer,
-                                      np.array([1, 2, 0], dtype=np.intp))
-
-        ordered, dexer = idx.sort_values(return_indexer=True, ascending=False)
-        self.assertTrue(ordered[::-1].is_monotonic)
-        self.assert_numpy_array_equal(dexer,
-                                      np.array([0, 2, 1], dtype=np.intp))
-
-    def test_round(self):
-
-        # round
-        dt = Timestamp('20130101 09:10:11')
```

Comment:
so this is something to add to the list.

I want to create a ``pandas/tests/scalars/test_timestamp.py`` (and test_timedelta.py). So anything specifically Timestamp related will go there.

this can be next iteration though.